### ISSUE TITLE
ZIL not being mined

### ISSUE THREAD

**Issue by rodrigobragamg** [TrailingStop/TT-Miner-release#29]
Hey again TS!
This is the Zil line:
cZ ZIL -PZ <zil15gnc34xk3v9vpk0e735fq0c7qzglrlue99pjwk>.<%WORKER_NAME%>:<passw>@br.crazypool.org:5005

I just returned and did not mine ZIL. It seems connected to the pool, but, didnt mine. Have any idea of what I can do?

5:30:42.616 Pool[ZIL-0]:   -*** Summary for: Zilliqa / ETHASH OC:[CL:1085,CO:+0,ML:10800-11600,MO:+0,PL:0,FAN:89-95] - Mixed values. Use -ocinfo for per device details
15:30:42.616 GPU[00-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.755 GHz-100%  MClk:5.001 GHz-0%  59°C  85%  214 W  0 H/s/W  23.41 GB   -
15:30:42.616 GPU[01-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.755 GHz-100%  MClk:5.001 GHz-0%  51°C  75%  194 W  0 H/s/W  9.549 GB   -
15:30:42.616 GPU[02-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.695 GHz-100%  MClk:5.001 GHz-0%  54°C  82%  168 W  0 H/s/W  9.549 GB   -
15:30:42.616 GPU[03-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.695 GHz-100%  MClk:5.001 GHz-0%  54°C  82%  97 W  0 H/s/W  9.549 GB   -
15:30:42.616 GPU[04-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.725 GHz-100%  MClk:5.001 GHz-0%  55°C  83%  166 W  0 H/s/W  9.549 GB   -
15:30:42.616 GPU[05-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.710 GHz-100%  MClk:5.001 GHz-0%  53°C  76%  153 W  0 H/s/W  9.549 GB   -
15:30:42.616 All[ZIL]:      -0 H/s [A0:R0:S0 0.0%]  992 W  0 H/s/W  uptime: 0 00:05:30  mining-time: 0 00:00:00  last: 00:00

**Comment by TrailingStop** [comment]
has there been a mining phase for zil?

**Comment by TrailingStop** [comment]
15:30:42.616 All[ZIL]: -0 H/s [A0:R0:S0 0.0%] 992 W 0 H/s/W uptime: 0 00:05:30 mining-time: 0 00:00:00 last: 00:00
5 minutes...

**Comment by rodrigobragamg** [comment]
Yeah.. Because I made some changes in the setting and restarted manually..

16:22:59.778 Pool[ZIL-0]:   -*** Summary for: Zilliqa / ETHASH OC:[CL:1085,CO:+0,ML:10400-11700,MO:+0,PL:0,FAN:90-95] - Mixed values. Use -ocinfo for per device details
16:22:59.778 GPU[00-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.785 GHz-100%  MClk:5.001 GHz-0%  58°C  85%  156 W  0 H/s/W  9.538 GB   -
16:22:59.778 GPU[01-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.710 GHz-100%  MClk:5.001 GHz-0%  57°C  0%  159 W  0 H/s/W  9.549 GB   -
16:22:59.778 GPU[02-ZIL]:   -0 H/s [A0:R0:S0 0.0%]  CClk:1.710 GHz-100%  MClk:5.001 GHz-0%  56°C  82%  170 W  0 H/s/W  9.549 GB   -
16:22:59.778 All[ZIL]:      -0 H/s [A0:R0:S0 0.0%]  485 W  0 H/s/W  uptime: 0 11:14:15  mining-time: 0 00:00:00  last: 00:00

**Comment by TrailingStop** [comment]
try without OC. always make sure that OC is not the problem first.
cZ ZIL -PZ .<%WORKER_NAME%>:@br.crazypool.org:5005

should be:
-cZ ZIL -PZ <wallet>.<%WORKER_NAME%>@br.crazypool.org:5005

**Comment by rodrigobragamg** [comment]
That ":" was the password... Well, can't mine ZIL without the proper OC. But if the settings are all correct, then I should wait for your testing/fixing.

**Comment by TrailingStop** [comment]
Can you please share the complete commandline you use - I will check here what the problem is.

**Comment by rodrigobragamg** [comment]
-oc-coreoff 295
-oc-core 1755 1755 1685 1695 1725 1705
-oc-mem 5001
-oc-fan 85 75 82 82 83 76
-cZ ZIL -PZ <zil15gnc34xk3v9vpk0e735fq0c7qzglrlue99pjwk>.<%WORKER_NAME%>@stratum+tcp://br.crazypool.org:5005
-oc-coreoffZ 0
-oc-coreZ 1085
-oc-memZ 11300 11600 10800 10900 11500 11100
-oc-fanZ 95 89 93 94 93 93

**Comment by rodrigobragamg** [comment]
P.S: I've tried changing the work_name for a fixed name, also the server with and without stratum+tcp://, etc.

**Comment by rodrigobragamg** [comment]
Were you able to reproduce the error?

**Comment by TrailingStop** [comment]
Hi. please change this:
-cZ ZIL -PZ .<%WORKER_NAME%>@stratum+tcp://br.crazypool.org:5005

to:
-cZ ZIL -PZ<wallet> .<%WORKER_NAME%>@br.crazypool.org:5005

**Comment by TrailingStop** [comment]
you can be sure it is you wallet address - if I remember correctly it start with zil......

**Comment by rodrigobragamg** [comment]
Yes, ofc.. The wallet is correct. But as you many know, github doesnt allow to be posted in here.

You can try the same command line with your own wallet. Did you do it?

**Comment by TrailingStop** [comment]
Hi-can you create a pool log so that i can see what's going on? -logpool.
Please send the log after you see the error.
Thanks.
